python/hail/__init__.py

```diff
@@ -27,6 +27,7 @@
 from .expr import *
 from .genetics import *
 from .methods import *
+from .experimental import *
```

Comment:
delete

Reply:
I would like to be able to call `hl.ld_score()`, not `hl.experimental.ld_score()`.

Reply:
I think experimental stuff needs to be explicitly such. @cseed thoughts?

Reply:
I'll just change it for now, but if there are going to be methods that sit in experimental long-term rather than a short stay en route to becoming a core method, I think users would prefer the former for regular use.